woocommerce/woocommerce
# Add e2e tests for virtual orders

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #44400.

This PR adds e2e tests for placing virtual orders. We're testing the following two scenarios:

1. Shopper can place a digital order when shipping is disabled.
2. Shopper can place a digital order when shipping is disabled, but Local Pickup is still enabled.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Verify that all CI tests are passing.
2. Verify that the e2e tests look great.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Add e2e tests for virtual orders

#### Comment 

</details>
